### PR TITLE
Pkg objects

### PR DESCRIPTION
**Pull request by richpauloo**
Main changes are to overwrite example data with pkg-objects in the README, and to update `get_data` to not depend on dplyr.

### CODE REVIEW

File: R/get_data.R [hunk 1]
```diff
@@ -32,21 +32,24 @@ get_data <- function(psid, start_date = NULL, end_date = NULL, analyte = NULL) {
...
          call. = FALSE)
   }
+
   url = paste0('https://sdwis.s3.us-west-1.amazonaws.com/', psid, '.csv.gz')
   d   = data.table::fread(url)
 
   # if present, filter by start and end dates and analytes
   if(!is.null(start_date)){
-    d <- dplyr::filter(d, SAMP_DATE >= start_date)
+    d <- d[d$SAMP_DATE >= start_date, ]
```
Comment (thesadie) [inline R/get_data.R]: could you pick a more informative variable name than `d`?

Comment (richpauloo) [reply R/get_data.R]: I'm thinking `s3_dat`. Also writing some tests for get_data(), and will add these to the PR.

**Review by ergz — APPROVED** [review]

File: data-raw/DATASET.R [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
-water_systems <- tibble(psid = c("103039", "103040"),
-                        water_system_name = c("A","B"),
-                        county = c("D","E"),
-                        zipcode = c(90201, 94601))
+# water_systems <- tibble(psid = c("103039", "103040"),
+#                         water_system_name = c("A","B"),
+#                         county = c("D","E"),
+#                         zipcode = c(90201, 94601))
+water_systems <- readr::read_rds("data-raw/water_systems.rds")
 usethis::use_data(water_systems, overwrite=TRUE)
```
Comment (richpauloo) [inline data-raw/DATASET.R:7]: Question: by changing this pkg-object and the two others in this script (analytes, psid_analyte) to the "real objects" will this break @ERGZ 's get_metadata.R tests?

File: R/get_data.R (hunk 1 above)
Comment (richpauloo) [reply R/get_data.R]: I've updated d to s3_dat, and added some more to the function, see commits below.

File: README.md [hunk 3]
```diff
@@ -2,18 +2,37 @@
 
 Query tool for SDWIS (Safe Drinking Water Information System)
 
-This package was developed as an entry to the CA Water Data Challenge 2020.  It demonstrates the utility of easy to use API wrappers to empower analysts to get more done with data with less effort. Currently the SDWIS database is inconvenient  to query and there is no public facing API. This wrapper grabs data hosted in an AWS s3 bucket so we can pretend there is already the infrastructure to support this type of development. 
+First, determine the PSIDs of interest.
 
-## Installation
+## Query metadata
```
Comment (thesadie) [inline README.md]: I don't want to merge your README as it stands into v0.1, it undos the edits Emanuel and I did.

File: data-raw/DATASET.R [hunk 4]
```diff
@@ -1,33 +1,27 @@
...
-                        zipcode = c(90201, 94601),
-                        stringsAsFactors = FALSE)
-
-usethis::use_data(water_systems, overwrite = TRUE)
+# water_systems <- tibble(psid = c("103039", "103040"),
+#                         water_system_name = c("A","B"),
+#                         county = c("D","E"),
+#                         zipcode = c(90201, 94601))
+water_systems <- readr::read_rds("data-raw/water_systems.rds")
+usethis::use_data(water_systems, overwrite=TRUE)
```
Comment (thesadie) [inline data-raw/DATASET.R]: style issue, missing spaces `overwrite = TRUE`

**Review by thesadie — CHANGES_REQUESTED** [review]
this is all good after you fix the overwrite of the README file

File: README.md (hunk 3 above)
Comment (richpauloo) [reply README.md]: opps. THANK you for catching this. The readme in pkg-objects and v0.1 branches are now identical.

**Review by thesadie — APPROVED** [review]

**Pull request merged**